### PR TITLE
test/store_test: add deferred test case setup to support explicit min…

### PR DESCRIPTION
**Pull request by ifed01**
…_alloc_size specification.

Fixes: http://tracker.ceph.com/issues/18857

Signed-off-by: Igor Fedotov <ifedotov@mirantis.com>

### CODE REVIEW

**Comment by ifed01** [comment]
[       OK ] ObjectStore/StoreTestSpecificAUSize.OnodeSizeTracking/2 (5939 ms)
[----------] 9 tests from ObjectStore/StoreTestSpecificAUSize (10143170 ms total)

[----------] Global test environment tear-down
[==========] 69 tests from 2 test cases ran. (10631240 ms total)
[  PASSED  ] 69 tests.

**Comment by xiexingguo** [comment]
retest this please

File: src/test/objectstore/store_test.cc [hunk 1]
```diff
@@ -100,6 +100,7 @@ bool sorted(const vector<ghobject_t> &in, bool bitwise) {
   return true;
 }
 
+
```
Comment (varadakari) [inline src/test/objectstore/store_test.cc]: extra line.

**Comment by tchaikov** [comment]
@ifed01 needs rebase.

**Comment by ifed01** [comment]
@tchaikov - rebased.

**Pull request merged**